### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by BartDeCaluwe**
This pr will add the following changes to the readme:

- add a setup and installation section
- add more info about the clusters
- update the setup instructions for a local cluster
- update the recommended way of tagging messages
- add information about antdesign and charts
- add information about tailwind
- fix the names of the different clusters

*List which issues are fixed by this PR. You must list at least one issue.*

## Checklist

- [ ] I used the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
- [ ] Unit tests have been created if a new feature was added.
- [ ] Translations have been added and extracted if a new feature was added.

### CODE REVIEW

**Comment by RuvendeGroote** [comment]
can you add an explanation to the classNames util? (maybe go over the code and look at what might be confusing later on. the classNames stuff for example is pretty non-standard so might be confusing)

**Comment by RuvendeGroote** [comment]
extract the .env.local.dist part to a separate section on how to configure the app. mention that further explanation can be found in the .env.local.dist file. add explanations in that file.

File: README.md [hunk 1]
```diff
@@ -50,9 +99,9 @@ Maximum connection rate per 10 seconds per IP: 40
 Maximum amount of data per 30 second: 100 MB
```
Comment (RuvendeGroote) [inline README.md]: maybe it's better to just explain different clusters and refer to https://docs.solana.com/cluster/rpc-endpoints. mention that pre-mainnet lives on devnet as well.

**Comment by RuvendeGroote** [comment]
mention typeform and how you can get past it.

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

**Comment by BartDeCaluwe** [comment]
> mention typeform and how you can get past it.

Is this still relevant? When we add the Markets, the typeform buttons will be moved to a less prominent space.

**Comment by RuvendeGroote** [comment]
> > mention typeform and how you can get past it.
> 
> Is this still relevant? When we add the Markets, the typeform buttons will be moved to a less prominent space.

no

**Review by RuvendeGroote — APPROVED** [review]

**Pull request merged**